lib/doorkeeper/engine.rb

```diff
@@ -5,7 +5,9 @@ class Engine < Rails::Engine
     end
 
     initializer "doorkeeper.locales" do |app|
-      app.config.i18n.fallbacks = [:en]
+      if app.config.i18n.fallbacks.blank?
+        app.config.i18n.fallbacks = [:en]
+      end
     end
```

Comment:
the current way should work, but I found this file load after the app's, so it may override app's config. 
that's should be avoid.